invenio_rdm_records/records/mappings/v6/rdmrecords/drafts/draft-v1.0.0.json

```diff
@@ -177,21 +177,24 @@
             },
             "locations" : {
               "properties" : {
-                "description" : {
-                  "type" : "text"
-                },
-                "place" : {
-                  "type" : "text"
-                },
-                "point" : {
+                "geometry": {
                   "properties" : {
-                    "lat" : {
-                      "type" : "double"
+                    "type" : {
+                      "type" : "keyword"
                     },
-                    "lon" : {
-                      "type" : "double"
+                    "coordinates" : {
+                      "type" : "float"
                     }
                   }
+                },
+                "place" : {
+                  "type" : "text"
+                },
+                "identifiers" : {
+                  "type" : "object"
```

Comment:
Minor: should this not specify subtypes as well?

Reply:
Well, it is of type object which means we can add more key-values (i.e. type and subtype). However, see comment below on identifiers.